client/src/api/api.js

```diff
@@ -1,9 +1,32 @@
 import axios from "axios";
 
 const MENTOR_API_URL = process.env.MENTOR_API_URL || "/mentor-api"; // eslint-disable-line no-undef
```

Comment:
Should we allow MENTOR_API_URL to be set the same way as MENTOR_VIDEO_HOST?

Reply:
Would be easy to add, but AFAIK there's no current use case for that feature? Unless there's a current use for the feature, just as soon wait until the need arises.